src/pages/apply.tsx

```diff
@@ -35,29 +45,58 @@ export default function Apply() {
         <link rel="icon" href="/favicon.ico" />
       </Head>
       <main className="flex h-svh max-h-svh flex-col items-center bg-primary-50">
-        <ApplyNavbar />
-        <div className="flex w-full flex-1 flex-grow items-center">
-          {/* TODO: make this responsive */}
-          <div
-            id="left-panel"
-            className="lg:w-xl flex h-full flex-grow flex-col space-y-8 bg-primary-100 p-9 pt-12 lg:max-w-xl"
-          >
-            <div className="space-y-2">
-              <h1 className="text-2xl font-medium">{heading}</h1>
-              <h2 className="text-sm text-slate-500">{subheading}</h2>
+        <div className="fixed bg-primary-50 sm:static">
+          <ApplyNavbar />
+          {isMobile && (
+            <div className="flex w-screen justify-around border-b px-3 md:invisible">
+              <div
+                className={`w-1/2 py-4 text-center ${formVisible ? "border-b border-primary-600 text-primary-600" : "border-b border-slate-400 text-slate-400"}`}
+                onClick={() => {
+                  setFormVisible(true);
+                  setPassportVisible(false);
+                }}
+              >
+                Form
+              </div>
```

Comment:
can we use buttons here? or better yet use https://ui.shadcn.com/docs/components/tabs and just restyle the TabsTrigger. This isn't accessible. Feel free to just make this a separate ticket though, we can do it later.

Reply:
jk tabs don't fit the use case, unless i want to re-render everything inside of the tab which seems stupid

Reply:
maybe it's not

Reply:
using tabs now